### PR TITLE
julia-1.7.0 ebuild

### PR DESCRIPTION
**Pull request by dzejej**
The ebuild for julia 1.7.0 with patches. Pacthes come from Arch linux
exept julia-turnoff-patchelf.patch.

### CODE REVIEW

**Comment by thesamesam** [comment]
Please add a link to the top of each patch for where you got it from (possibly more links if there's any upstream commits / bugs to reference). Also add an explanation of why it's needed/what it does where it's not obvious.

Please also use repoman commit / pkgdev commit to get the commit message right and include [signoff](https://wiki.gentoo.org/wiki/Gentoo_git_workflow#Signoff). Thank you!

**Comment by anaveragehuman** [comment]
Hi! I usually submit PRs for this package but have been busier than usual this release cycle and was stuck on some strange part of their build system. On top of Sam's suggestions please also make sure the build is successful with `FEATURES=network-sandbox`.

**Comment by dzejej** [comment]
Sorry for pushing julia-1.7.0  twice.

File: dev-lang/julia/files/julia-hardcoded-libs-1.7.0.patch [hunk 1]
```diff
@@ -1,3 +1,6 @@
+This pacth comes from:
```
Comment (mattst88) [inline dev-lang/julia/files/julia-hardcoded-libs-1.7.0.patch:1]: typo: patch

File: dev-lang/julia/files/julia-llvm13-fix_A.patch [hunk 2]
```diff
@@ -1,3 +1,8 @@
+This patch is from julialang repository:
+ https://github.com/JuliaLang/julia/commit/677ce6d3.patch
+but reference comes form ARCH Linux juia package
```
Comment (mattst88) [inline dev-lang/julia/files/julia-llvm13-fix_A.patch:3]: typos: from, julia

File: dev-lang/julia/files/julia-llvm13-fix_B.patch [hunk 3]
```diff
@@ -1,3 +1,8 @@
+This patch is from julialang repository:
+ https://github.com/JuliaLang/julia/commit/47f9139e.patch
+but reference comes form ARCH Linux juia package
```
Comment (mattst88) [inline dev-lang/julia/files/julia-llvm13-fix_B.patch:3]: typos: from, julia

File: dev-lang/julia/files/julia-make-install-no-build.patch [hunk 4]
```diff
@@ -1,3 +1,6 @@
+This pacth comes from
```
Comment (mattst88) [inline dev-lang/julia/files/julia-make-install-no-build.patch:1]: typo: patch

File: dev-lang/julia/files/julia-turnoff-patchelf.patch [hunk 5]
```diff
@@ -1,3 +1,6 @@
+This patch turn off patching the binary files by patchelf.
```
Comment (mattst88) [inline dev-lang/julia/files/julia-turnoff-patchelf.patch:1]: mistake: 'turn' should be 'turns'

File: dev-lang/julia/files/julia-turnoff-patchelf.patch [hunk 6]
```diff
@@ -1,3 +1,6 @@
+This patch turn off patching the binary files by patchelf. 
+On my system (gentoo stable and updated) with patching causes
```
Comment (mattst88) [inline dev-lang/julia/files/julia-turnoff-patchelf.patch:2]: I don't know what this is saying. Did the sentence end too early?

File: dev-lang/julia/julia-1.7.0.ebuild [hunk 7]
```diff
@@ -62,6 +62,7 @@ DEPEND="${RDEPEND}
 # https://github.com/archlinux/svntogit-community/tree/7d240cefc127c1a8060316e9296445ef02a06a7b/trunk
 PATCHES=(
 	"${FILESDIR}/${PN}"-1.4.0-no_symlink_llvm.patch
+	# Allow use LLVM 13
```
Comment (mattst88) [inline dev-lang/julia/julia-1.7.0.ebuild:65]: I think this comment is kind of unnecessary.

**Review by mattst88 — CHANGES_REQUESTED** [review]
Please give patches appropriate names. E.g. instead of `dev-lang/julia: julia-1.7.1` it could be `dev-lang/julia: Version bump to 1.7.1`. Instead of `dev-lang/julia: ebuild for julia 1.7.0` it could be `dev-lang/julia: Document source of patches` or something like that.

**Pull request closed without merging**